### PR TITLE
Change how apt keys are downloaded

### PR DESCRIPTION
**Pull request by jaapmarcus**

### CODE REVIEW

**Review by ScIT-Raphael — CHANGES_REQUESTED** [review]
Shouldnt we also include the changes in existing hestia versions over the upgrade script?

File: install/upgrade/versions/1.5.3.sh [hunk 1]
```diff
@@ -19,4 +19,70 @@ upgrade_config_set_value 'UPGRADE_UPDATE_WEB_TEMPLATES' 'false'
...
+            echo "   [ * ] Apache2"
+            echo "deb [arch=$ARCH signed-by=/usr/share/keyrings/apache2-keyring.gpg] https://packages.sury.org/apache2/ $codename main" > $apt/apache2.list
+            curl -s https://packages.sury.org/apache2/apt.gpg | gpg --dearmor | tee /usr/share/keyrings/apache2-keyring.gpg >/dev/null 2>&1
+        fi
+    fi
+    if [ -f "$apt/mariadb.list" ]; then
+        rm  $apt/mariadb.list 
+        echo "   [ * ] MariaDB"
+        echo "deb [arch=$ARCH signed-by=/usr/share/keyrings/mariadb-keyring.gpg] https://mirror.mva-n.net/mariadb/repo//$VERSION $codename main" > $apt/mariadb.list
+        curl -s https://mariadb.org/mariadb_release_signing_key.asc | gpg --dearmor | tee /usr/share/keyrings/mariadb-keyring.gpg >/dev/null 2>&1
```
Comment (kpapad904) [inline install/upgrade/versions/1.5.3.sh]: line 71 should be:

https ://mirror.mva-n.net/mariadb//repo/$mariadb_v/$VERSION $codename main

**Comment by jaapmarcus** [comment]
Changed it

**Comment by kpapad904** [comment]
Actually it seems that the correct syntax for install/upgrade/versions/1.5.3.sh should be:

`https ://mirror.mva-n.net/mariadb/repo/$mariadb_v/$os $codename main
`

Note: Currently there is an inconsistency between the install and upgrade scripts in how they derive the variables $VERSION and $os …
```
VERSION="$(lsb_release -s -r)"
VERSION=‘debian’
```

For consistency reasons $VERSION should be the same in all 3 scripts (install/hst-install-debian.sh install/hst-install-ubuntu.sh and install/upgrade/versions/1.5.3.sh)

**Comment by jaapmarcus** [comment]
Don't know why both VERSION and OS are hardcoded... 

Made some minor changes

**Review by ScIT-Raphael — APPROVED** [review]

**Pull request merged**